websites/templates/mail/preview_publish_failure/body.html

```diff
@@ -0,0 +1,19 @@
+{% extends "mail/email_base.html" %}
+
+{% block content %}
+<tr>
+  <td style="padding: 20px;">
+    <table role="presentation" cellspacing="0" cellpadding="0" border="0" width="100%">
+      <tr>
+        <td style="padding: 20px; font-family: sans-serif; font-size: 15px; line-height: 20px; color: #555555;">
+          <p style="margin: 0 0 10px;">Dear {{ user.name }},</p>
+          <p style="margin: 0 0 10px;">
+            The {{ version }} version of your site <a href="{{ site.url }}">{{ site.title }}</a> failed to complete.
```

Comment:
Should the wording be some like "failed to update" instead of "failed to complete" since people outside the team don't know there is a concourse build that needs to happen?